### ISSUE TITLE
[Feature Request] Button to smooth scroll and custom keybinding

### ISSUE THREAD

**Issue by parointi** [xgi/houdoku#64]
1. In vertical view, the only way to move between pages is to scroll with the mousewheel. It would be great if there were keyboard buttons that let you smoothly scroll up and down.
2. Most obvious keys would be up and down arrow keys. However, I would prefer `j` for down and `k` for up (habits from vim). The ability to customize keybinding for all the reader functionalities would be amazing.

Technically these are two feature requests. Let me know if it's appropriate to split into two separate issues.

**Comment by xgi** [comment]
Yeah, the page movement on the vertical view is something I wanted to have when the view was originally made, but I encountered tech difficulties while implementing it. My idea was just to have the left/right arrows move down to the next image, the same way you change pages on the horizontal view. Last time I looked at it, there wasn't a straightforward way to do that with the current way things are rendered. I have it on my list to take another look soon, but I can't promise I'll have a solution.

(it would also be good if the page # field updated automatically when you scroll, but that's even more complicated)

I can do custom keybindings, though! That's just a QOL thing I hadn't gotten around to yet, but I can work on it next.
